infra/scripts/common/build.sc

```diff
@@ -10,27 +10,38 @@ private def directoryToProjectName(directoryName: String) = {
   segments.head + segments.tail.map(_.capitalize).mkString
 }
 
-def buildStack(projects: Seq[String], registry: Registry,
-               localRepo: Boolean = false, skipTests: Boolean = true, skipPublish: Boolean = true)
-              (implicit progressBar: ProgressBar): Unit = {
+def appsInParadigm(implicit stackType: StackType, steps: StepDefinitions): Seq[(String, Int)] = {
+  apps.map { a =>
+    s"${a._1}-${stackType.paradigm}" -> a._2
+  } ++
+  backingServices ++
+  ( if(steps.gatling) Seq(gatling -> -1) else Nil )
+}
+
+def buildStack(projects: Seq[String], publishOpts: PublishOptions)
+              (implicit progressBar: ProgressBar, steps: StepDefinitions): Unit = {
   implicit val codebasePath = pwd / "codebase"
-  progressBar stepInto "Build"
+  progressBar.stepInto("Build")
+  progressBar.show("SBT build in progress..")
 
-  projects foreach { directory =>
+  val sbtParams = projects.map { directory: String =>
     val project = directoryToProjectName(directory)
 
-    if(!skipTests) {
-      progressBar show s"Testing $project..."
-      % sbt(s"-Ddocker.registry.host=${registry.value}", "coverageOff", s"$project/test")
-    }
+    Seq(
```

Comment:
combining all Sbt tasks and running them in a single SBT run speeds up process significantly